docs/guides/cloud/branch-review.mdx

```diff
@@ -83,15 +62,29 @@ If the selected branch has a PR associated and a PR # passed via CI, there is an
 
 ### View Options
 
-"Review" is the default view of the **Branches** tab for projects with linked GitHub repositories via our GitHub integration.
+"Overview" is the default view of the **Branches** tab for projects with linked GitHub repositories via our GitHub integration.
 
-"Runs" provides a list filtered by the selected branch for projects without linked repositories or those with an unknown branch. An unkown branch is one in which Cypress does not have access to the local `.git` directory and cannot determine a branch for a run.
+:::note
+
+1. Callouts for the latest available Branch Review or a call to action to create a Pull Request in order to compare test run results.
+
+2. Click "Review changes" for the full Branch Review.
+
+3. Click any test status to reveal those changes.
+
+4. Latest runs of the current branch for contextual guidance on the state of its test runs.
+
+:::
 
 <DocsImage
-  src="/img/guides/cloud/branch-review/branch-review-views.jpg"
+  src="/img/guides/cloud/branch-review/views3.png"
   alt="Displaying different views for branches"
 />
 
+"Review" will lead to the latest available Branch Review.
+
+"Runs" provides a list filtered by the selected branch for projects without linked repositories or those with an unknown branch. An unkown branch is one in which Cypress does not have access to the local `.git` directory and cannot determine a branch for a run.
```

Comment:
Runs provides a list of all runs on the current branch (in all cases). Nothing to do with linking repos or unknown branches.